example/worker.js

```diff
@@ -1,24 +1,24 @@
 // const ZB = require('zeebe-node');
-const ZB = require('../dist')
+const ZB = require("../dist")
```

Comment:
Any reason we're using quotes here?

Reply:
I think my system has some circular dependency in the linting tools. The example js and the docs get linted and fixed twice. The only way that I can get them in is via `--no-verify`. If we have docs build automatically it will do away with that issue, and I'll debug the js part.